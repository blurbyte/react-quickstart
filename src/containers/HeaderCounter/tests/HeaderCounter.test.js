import React from 'react';
import renderer from 'react-test-renderer';

import { HeaderCounter } from '../index';
jest.mock('../Counter', () => 'Counter');

describe('<HeaderCounter />', () => {
  it('should render correctly', () => {
    // props init
    const props = {
      counter: 10,
      actions: {
        testAction: jest.fn()
      }
    };

    // component init
    const component = renderer.create(<HeaderCounter {...props} />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
