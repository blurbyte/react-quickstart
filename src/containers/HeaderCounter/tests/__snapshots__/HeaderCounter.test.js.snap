// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderCounter /> should render correctly 1`] = `
<Counter
  counter={10}
/>
`;
